waku/waku_peer_exchange/rpc.nim

```diff
@@ -8,6 +10,50 @@ type
   PeerExchangeResponse* = object
     peerInfos*: seq[PeerExchangePeerInfo]
 
+  PeerExchangeResponseStatusCode* {.pure.} = enum
+    UNKNOWN = uint32(000)
+    SUCCESS = uint32(200)
+    BAD_REQUEST = uint32(400)
+    BAD_RESPONSE = uint32(401)
+    TOO_MANY_REQUESTS = uint32(429)
+    SERVICE_UNAVAILABLE = uint32(503)
+    DIAL_FAILURE = uint32(599)
```

Comment:
We are having some repeated codes' definitions in multiple modules. Maybe in an upcoming PR we can unify those definitions in a shared module to avoid possible discrepancies if we change sth